packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,60 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- Add a new `setProvider` method to `AssetsContractController` ([#4397](https://github.com/MetaMask/core/pull/4397))
+  - Replaces the removed `provider` setter method, and widens the `provider` function parameter type from `Provider` to `Provider | undefined`.
+- Export `TokenBalancesControllerState` type ([#4535](https://github.com/MetaMask/core/pull/4535))
+  - This was defined but not exported in v34.0.0.
+
+### Changed
+
+- **BREAKING:** The type of `SINGLE_CALL_BALANCES_ADDRESS_BY_CHAINID` is narrowed from `Record<Hex, string>` to the const-asserted literal properties of the `SINGLE_CALL_BALANCES_ADDRESS_BY_CHAINID` object ([#4397](https://github.com/MetaMask/core/pull/4397))
+  - The index signature is restricted to the union of the enum keys of `SupportedTokenDetectionNetworks`.
+  - The property value type is restricted to the type union of the addresses defined in the object.
+  - The object type is constrained by `Record<Hex, string>` using the `satisfies` keyword.
+- **BREAKING:** Convert the `BalanceMap` type from an `interface` into a type alias ([#4397](https://github.com/MetaMask/core/pull/4397))
+  - Type aliases have an index signature of `string` by default, and are compatible with the `StateConstraint` type defined in the `@metamask/base-controller` package.
+- **BREAKING:** `getIpfsCIDv1AndPath`, `getFormattedIpfsUrl` are now async functions ([#3645](https://github.com/MetaMask/core/pull/3645))
+- Add `immer` `^9.0.6` as a new dependency ([#3645](https://github.com/MetaMask/core/pull/3645))
+- Bump `@metamask/abi-utils` from `^2.0.2` to `^2.0.3` ([#3645](https://github.com/MetaMask/core/pull/3645))
+- Bump `@metamask/approval-controller` from `^7.0.1` to `^7.0.2` ([#4544](https://github.com/MetaMask/core/pull/4544))
+- Bump `@metamask/base-controller` from `^6.0.0` to `^6.0.2` ([#4517](https://github.com/MetaMask/core/pull/4517), [#4544](https://github.com/MetaMask/core/pull/4544))
+- Bump `@metamask/controller-utils` from `^11.0.1` to `^11.0.2` ([#4544](https://github.com/MetaMask/core/pull/4544))
+- Bump `@metamask/keyring-controller` from `^17.1.0` to `^17.1.1` ([#4527](https://github.com/MetaMask/core/pull/4527))
```

Comment:
This gets bumped in this PR:

``` suggestion
- Bump `@metamask/keyring-controller` from `^17.1.0` to `^17.1.2` ([#4527](https://github.com/MetaMask/core/pull/4527), [#4548](https://github.com/MetaMask/core/pull/4548))
```

Reply:
```suggestion
```

- See https://github.com/MetaMask/core/pull/4556